app/models/package.rb

```diff
@@ -26,6 +26,7 @@ class Package < ActiveRecord::Base
   before_save :save_inventory_number, if: :inventory_number_changed?
   before_save :update_set_relation, if: :stockit_sent_on_changed?
   after_commit :update_set_item_id, on: :destroy
+  after_touch {update_client_store :update unless Rails.env.test? }
```

Comment:
I'd prefer to remove the `unless Rails.env.test?` and stub out push updates in the test, globally.